Sources/SpeziOnboarding/OnboardingView.swift

```diff
@@ -39,6 +41,32 @@ import SwiftUI
 ///     // Action that should be performed on pressing the "Continue" button ...
 /// }
 /// ```
+///
+///  In implementation, you can treat the titleView, contentView, and actionView as regular views. However, to simplify things, you can also use the built-in OnboardingTitleView and built-in OnboardingActionsView, as demonstrated below.
```

Comment:
Feel free to format this entire block of added content within an `### Usage` section